lib/irc/ClientPool.js

```diff
@@ -248,6 +248,17 @@ ClientPool.prototype.totalReconnectsWaiting = function (serverDomain) {
     return 0;
 }
 
+ClientPool.prototype.getNickUserIdMappingForChannel = function(server, channel) {
```

Comment:
How fast is this function on a channel with tens of thousands of nicks? This seems like a good thing to have a live cache for.

Reply:
I'm thinking that...I guess we only need to invalidate this on join.